src/structures/WelcomeChannel.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const Base = require('./Base');
+const Emoji = require('./Emoji');
+
+/**
+ * Represents a channel link in a guild's welcome screen.
+ */
```

Comment:
```suggestion
/**
 * Represents a channel link in a guild's welcome screen.
 * @extends {Base}
 */
```
This class also needs to be added to the exports